src/main/webapp/register.html

```diff
@@ -51,27 +51,33 @@ <h5 class="text-center lead">Fill in the following fields below.</h5>
           <div class="col-md-12 mb-3">
             <label for="user-email">Email</label>
             <!-- TODO: replace with user's actual email -->
-            <input type="text" readonly class="form-control" id="user-email" value="tess@gmail.com">
+            <input type="text" readonly class="form-control" id="user-email" value="tess@gmail.com" name="user-email">
           </div>
         </div>
         <div class="form-row">
           <div class="col-md-6 mb-3">
             <label for="first-name-field">First name</label>
-            <input type="text" class="form-control" id="first-name-field" placeholder="First name" value="" required>
+            <input type="text" class="form-control" id="first-name-field" placeholder="First name" value="" required name="first-name">
+            <div class="valid-feedback">
+              Looks good!
+            </div>
```

Comment:
Why did we put these back? Were the green check marks not enough?

Reply:
Ah it's because the other PR in which l changed these got closed.